templates/bot/js/notification-function-base/README.md

```diff
@@ -46,238 +40,78 @@ This section walks through the generated code. The project folder contains the f
-const myStorage = new MyStorage(...);
-
-// initialize ConversationBot with notification enabled and customized storage
-const bot = new ConversationBot({
-    // The bot id and password to create BotFrameworkAdapter.
-    // See https://aka.ms/about-bot-adapter to learn more about adapters.
-    adapterConfig: {
-        appId: process.env.BOT_ID,
-        appPassword: process.env.BOT_PASSWORD,
-    },
-    // Enable notification
-    notification: {
-        enabled: true,
-        storage: myStorage,
-    },
-});
-```
-
-**[This Sample](https://github.com/OfficeDev/TeamsFx-Samples/blob/ga/adaptive-card-notification/bot/src/storage/blobsStorage.ts)** provides a sample implementation that persists to Azure Blob Storage.
-
-## Add command and responses to your application
-
-The command and response feature adds the ability for your application to "listen" to commands sent to it via a Teams message. A response (in the form of an Adaptive Card) is sent back to Teams. You can register multiple commands and have individual responses for each command.
+## Extend the notification bot template
 
-To add the command and response feature:
+There are few customizations you can make to extend the template to fit your business requirements.
 
-1. Go to `src\internal\initialize.js`
-2. Update the `conversationBot` initialization to enable command-response feature: 
-   ![enable-command](https://user-images.githubusercontent.com/10163840/165430233-04648a2a-d637-41f0-bb17-b34ddbd609f7.png)
-3. Follow [these instructions](https://aka.ms/teamsfx-command-response#How-to-add-more-command-and-response) to add commands to your application.
+1. [Step 1: Customize the trigger point from event source](#step-1-customize-the-trigger-point-from-event-source)
+2. [Step 2: Customize the notification content](#step-2-customize-the-notification-content)
+3. [Step 3: Customize where notifications are sent](#step-3-customize-where-notifications-are-sent)
 
-## Add adaptive card actions
-The Adaptive Card action handler feature enables the app to respond to adaptive card actions that triggered by end users to complete a sequential workflow. When user gets an Adaptive Card, it can provide one or more buttons in the card to ask for user's input, do something like calling some APIs, and then send another adaptive card in conversation to response to the card action.
+### Step 1: Customize the trigger point from event source
 
-To add adaptive card actions to notification bot, you can follow the steps [here](https://aka.ms/teamsfx-card-action-response#add-more-card-actions).
+By default Teams Toolkit scaffolds a single `restify` entry point in `src/index.js`. When a HTTP request is sent to this entry point, the default implementation sends a hard-coded Adaptive Card to Teams. You can customize this behavior by customizing `src/index.js`. A typical implementation might make an API call to retrieve some events and/or data, and then send an Adaptive Card as appropriate.
```

Comment:
This is the way to add trigger for restify-based notification bot. For azfunc-based one, it's totally another story.

Reply:
Updated